### DISCUSSION TITLE
NR 599 midterm questions

### DISCUSSION THREAD

**Discussion by D41265729** [community/community#114314] [GitHub Education]
### Hi everyone,

Hi , I need NR 599 midterm questions

**Comment by samus-aran** [comment]
Hi @D41265729 ,
The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was flagged and removed because it violated our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. Posting screenshots of test questions without context will likely discourage engagement and responses.

If this content is posted again, your Community account will be subject to immediate and indefinite suspension.
For additional guidance, please review our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service).

Thank you!
